MainHospital/Views/submit.php

```diff
@@ -10,19 +11,17 @@
   }
    if((!isset($_SESSION['usermain'])) && (!isset($_COOKIE['usermain'])))
   {
-	  $home_url = 'http://' . $_SERVER['HTTP_HOST'] . dirname($_SERVER['PHP_SELF']) . '/login-main.php';
+	  $home_url = 'http://' . $_SERVER['HTTP_HOST'] . dirname($_SERVER['PHP_SELF']) . '/Main Hospital/Views/login-main.php';
   header('Location: ' . $home_url);
   }
   if (isset($_SESSION['usermain'])) {
 
   }
 $id = $_GET['id'];
-require_once("connectvars.php");
 
-$dbc = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME)
+$dbc = mysqli_connect('localhost','root','','healthcare')
```

Comment:
@ankitjain28may  , @aakarshgoel96  I think its coming from the config file , can not write directly.